packages/block-editor/src/components/block-has-dot/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Fill } from '@wordpress/components';
+
+function BlockHasDot( {
+	parentReusableBlockHasEdits,
+	selectedReusableBlockHasEdits,
```

Comment:
This is another place where it'd be good to make this more generic, so it isn't just a feature that only reusable blocks can take advantage of.

I don't think these props are needed, since the reusable block code itself can optionally render these. It might require splitting this into two separate components (one for the parent block selector, one for the current block), but something like the following code in the reusable block edit function should achieve it:
```jsx
( isSelected && hasEdits && {
    <SelectedBlockUnsavedChangesIndicator />
 } )
( hasChildBlockSelected && hasEdits && {
   <ParentBlockSelectorUnsavedChangesIndicator />
 } )
 ```

Reply:
The props were used to update the classes of `BlockContextualToolbar` and the `BlockParentSelector`.

Is there some other way to conditionally load styles for `BlockContextualToolbar` and `BlockParentSelector` when the `BlockUnsavedChangesIndicator` is mounted?